packages/eslint-plugin/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@swisspost/design-system-eslint-plugin",
```

Comment:
I would propose to call the package 
- `@swisspost/design-system-eslint-migrations` 
- `@swisspost/design-system-eslint-plugin-migrations`
- `@swisspost/eslint-plugin-design-system-migrations`

This would better communicate the purpose of the package and prevent naming conflicts if for example we decide to have a default ruleset eslint plugin shared with other projects. What do you think?

Reply:
We build the package for migration purposes but it is now ready to export any eslint config or plugin. This is the biggest advantage of the ESLint flat config!
Therefore, I would simply call the package `@swisspost/design-system-eslint`.

Then we could rethink the name of the plugin itself: https://github.com/swisspost/design-system/blob/077bb70019ab0d427543eec41d9cd6ed3020a7fa/packages/eslint-plugin/src/index.ts#L13
But do you think it's an issue to have the rules installed in a project at all times? For example, the rule I created notifies you when you use the `.btn-rg` class, this might be useful even outside of a migration phase, don't you think?

Reply:
@alizedebray Most of the packages I saw, are names like this `@namespace/eslint-plugin`.
But I really support to not use `migration` in it's name!

Reply:
The `@namespace/eslint-plugin` packages are usually used for classic config, the flat config packages are for example `typescript-eslint` and `angular-eslint`. We could use `swisspost-eslint`, but I have a preference for `@swisspost/design-system-eslint` to align with our naming convention.

Reply:
@gfellerph grouping the rules to a specific migration might not be necessary as the rules will simply differs from one version of the package to the other: `@swisspost/design-system-eslint@8` exports rules to migrate from v7 to v8, but `@swisspost/design-system-eslint@9` exports rule to migrate from v8 to v9.

Reply:
@alizedebray This would mean, we would remove existing rules, as soon as we start working on a next version of this package!?

The more I think about it, the more this makes sense!
Because else, we would still provide rules to migrate from v10 to v11, which have been created to migrate from v8 to v9.
And this, would not make any sense and could even produce wrong output.